src/pages/community.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+
+import styles from './index.module.scss';
+import clsx from 'clsx';
+
+function Community() {
+  return (
+    <Layout title="Community">
+      <div className={clsx('subtron', styles.subtron)}>
```

Comment:
If this subtron is meant to be reusable, we could potentially move it to its own component.